UrTorstigaStrupar/SornaiGusha.tex

```diff
@@ -4,13 +4,13 @@
 \beginverse*						
 Sörnai gusha nietu Molotova
 sörnai gusha herba Moskova.
-:,: Njet, njet bonimal votvot risubuska,
+:,: Njet, njet bonimai votvot risubuska,
 dara zeva votvot harasoo. :,:
 \endverse
 
 \beginverse				
-Isa Stalin ja äiti Grutzheva,
-iski silmää gini hibragas.
+Isä Stalin ja äiti Hrushtsheva,
```

Comment:
Jag tror det är "Isa" här för att det ska låta mer ryskt i uttalet. Finns det exempel i andra sångböcker?

Reply:
På dom här sidorna skriver dom Isä och inte isa: http://laulu.wiki/id/33 http://wiki.digit.fi/UrhoMatti:S%C3%B6rnai_gusha

Reply:
"Isä" it is, then!